torch/_inductor/ir.py

```diff
@@ -3718,13 +3716,15 @@ def __init__(
                     # repeat_interleave(const at::Tensor & repeats, c10::optional<int64_t> output_size=c10::nullopt)
                     # repeat_interleave(const at::Tensor & self, int64_t repeats,
                     #       c10::optional<int64_t> dim=c10::nullopt, c10::optional<int64_t> output_size=c10::nullopt)
-                    self.kernel = (
-                        f"at::_ops::{kernel.__name__.replace('.default', '')}::call"
-                        if kernel._overloadname == "default"
-                        else f"at::_ops::{kernel.__name__.replace('.', '_')}::call"
-                    )
+                    # self.kernel = (
+                    #     f"at::_ops::{kernel.__name__.replace('.default', '')}::call"
+                    #     if kernel._overloadname == "default"
+                    #     else f"at::_ops::{kernel.__name__.replace('.', '_')}::call"
+                    # )
+                    self.kernel = f"at::{op_base_name}"
                     schema = kernel._schema
                 else:
+                    # For torch._ops.HigherOrderOperator
```

Comment:
Uh, so we generate an `at::cond` for HigherOrderOperator? That doesn't sound right (there isn't an at::cond)

Reply:
indeed, this path shouldn't be needed anymore. 

HigherOrderOps should be handled in this branch. 
`elif isinstance(kernel, torch._ops.HigherOrderOperator):`

Reply:
This else branch has been removed.